content/en/blog/_posts/2022-12-16-statefulset-migration.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: blog
+title: "Kubernetes 1.26: StatefulSet Migration"
```

Comment:
If we do keep the MCS aspect of this article, then maybe this:
```suggestion
title: "Kubernetes 1.26: StatefulSet Numbering Helps With Stateful Workload Migration"
```

The post-release articles follow a pattern: they introduce a new feature and explain how it's useful. We shouldn't omit the first bit, the introduction. Imagine you're at a SIG Apps meeting and you're explaining how this new feature, not yet implemented, will benefit end users. What would you say?

Reply:
Updated the title to reflect the benefit of this feature, and goal of the tutorial